client/src/Components/SubmitScore/SubmitScore.js

```diff
@@ -1,62 +1,126 @@
 import styled from 'styled-components/macro';
 import Button from '../Button/Button';
 import Header from '../Header/Header';
+import { useState } from 'react';
+import {
+  useNewHighScore,
+  useSetNewHighScore,
+} from '../../Services/Context';
 
 export default function SubmitScore({
   playerScore,
+  scoreData,
 }) {
-  function handleClick() {
-    console.log('eellooo');
+  console.log(scoreData);
+  const [
+    isScoreSubmitted,
+    setIsScoreSubmitted,
+  ] = useState(false);
+  const [playerInfo, setPlayerInfo] = useState({
+    playerName: '',
+    score: playerScore,
+  });
+  const newHighScore = useNewHighScore();
+  const setNewHighScore = useSetNewHighScore();
+
+  function handleChange(event) {
+    const fieldValue = event.target.value;
+    console.log(fieldValue);
+    setPlayerInfo({
+      ...playerInfo,
+      [event.target.name]: fieldValue,
+    });
+  }
+
+  function onSubmit(event) {
+    event.preventDefault();
+    fetch('http://localhost:4000/api/scores', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(playerInfo),
+    })
+      .then((data) => data.json())
```

Comment:
You could consider to outsource the fetch here into a service function (e.g. fetchScores) to make the onSubmit function more readable